lib/iris/tests/unit/fileformats/pp/test_PPField.py

```diff
@@ -35,7 +35,18 @@
     ("lbext", (19,)),
     ("lbpack", (20,)),
     ("dummy5", (21, 37)),
-    ("lbuser", (38, 39, 40, 41, 42, 43, 44,)),
+    (
+        "lbuser",
+        (
+            38,
+            39,
+            40,
+            41,
+            42,
+            43,
+            44,
+        ),
+    ),
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?